lib/web/apiserver.go

```diff
@@ -2086,25 +2114,19 @@ func (h *Handler) siteSessionsGet(w http.ResponseWriter, r *http.Request, p http
 		return nil, trace.Wrap(err)
 	}
 
-	sessions, err := clt.GetSessions(apidefaults.Namespace)
+	trackers, err := clt.GetActiveSessionTrackers(r.Context())
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
 
-	// DELETE IN: 5.0.0
-	// Teleport Nodes < v4.3 does not set ClusterName, ServerHostname with new sessions,
-	// which 4.3 UI client relies on to create URL's and display node inform.
-	clusterName := p.ByName("site")
-	for i, session := range sessions {
-		if session.ClusterName == "" {
-			sessions[i].ClusterName = clusterName
-		}
-		if session.ServerHostname == "" {
-			sessions[i].ServerHostname = session.ServerID
+	sessions := make([]session.Session, 0, len(trackers))
+	for _, tracker := range trackers {
+		if tracker.GetSessionKind() == types.SSHSessionKind {
+			sessions = append(sessions, trackerToLegacySession(tracker, site.GetName()))
 		}
 	}
 
-	return siteSessionsGetResponse{Sessions: sessions}, nil
+	return siteSessionsGetResponse{sessions}, nil
```

Comment:
Did we change the JSON format of the output by doing this?

Reply:
We do not, the struct is still defined the same. This is just how you initialize single field Go structs without naming the field.